_dev/apps/ui/src/store/modules/accounts/actions.ts

```diff
@@ -69,6 +70,17 @@ export default {
     dispatch(ActionsTypes.SEND_GMC_INFORMATION_TO_SHOP, {
       id: selectedAccount.id,
     });
+    const getModulesStatus = await dispatch(`app/${ActionsTypesApp.GET_MODULES_VERSIONS}`, 'psxmarketingwithgoogle', {root: true});
+
+    if (getModulesStatus?.hooks) {
+      Object.entries(getModulesStatus.hooks)
+        .forEach(([key, value]) => {
+          if (value === false) {
+            dispatch(`app/${ActionsTypesApp.TRIGGER_REGISTER_HOOK}`, key, {root: true});
+          }
+        });
+    }
+
     commit(MutationsTypes.SAVE_GMC, selectedAccount);
```

Comment:
What do you think about storing SAVE_GMC before the check of hooks, so the interface can be already updated with the connected account while we check the hooks?